test/games/strategy/triplea/delegate/MoveDelegateTest.java

```diff
@@ -1047,8 +1047,8 @@ public void testReloadTransportAfterRetreatAllied() {
     retreatingLandUnits.addAll(karelia.getUnits().getMatches(Matches.isUnitAllied(russians, m_data)));
     final Integer retreatingLandSizeInt = retreatingLandUnits.size();
     // Get the defending land units that and their number
-    final List<Unit> defendingLandUnits = new ArrayList<>();
     retreatingLandUnits.addAll(karelia.getUnits().getMatches(Matches.isUnitAllied(british, m_data)));
+    final List<Unit> defendingLandUnits = new ArrayList<Unit>();
```

Comment:
- It's not just the few minutes
- We incur yet another round trip on the PR
- It does not matter if it is never fixed. One of these hanging around out of the 2500 removed is not terribly significant
- I disagree we'll never come back to it, I changed them all once, can easily do more passes. That is actually how I fixed them here, by re-running the auto cleanup.